eng/deploy.yaml

```diff
@@ -44,16 +88,15 @@ stages:
 - stage: deploy
   displayName: Deploy
   dependsOn:
-  - build
   - predeploy
   variables:
-  - group: ${{ parameters.VariableGroup }}
+  - group: $(VariableGroup)
```

Comment:
I don't think variable groups work like this either.

Reply:
Changed variable groups inclusion to be global